projects/1.19/assets/bellsandwhistles/bellsandwhistles/lang/zh_cn.json

```
@@ -0,0 +1,43 @@
+{
+  "block.bellsandwhistles.andesite_grab_rails": "安山岩梯杆",

```

```suggestion
  "block.bellsandwhistles.andesite_grab_rails": "安山梯杆",
```
参照机械动力本体
这里的合成材料应该是安山合金